### DISCUSSION TITLE
Horizontal lazy loading in gatsby-plugin-image

### DISCUSSION THREAD

**Discussion by firflant** [gatsbyjs/gatsby#33230] [Help]
Hey! I build a site that uses gatsby-plugin-image. Most of the images loads fine, but there is a problem with cards located in carousel gallery. Images that appears to viewport from right edge of a screen (on slide) are not lazy loaded properly and they are being hydrated after the slide transition starts - it is too late and there is an ugly blink effect.

Is there any method to disable lazy loading for gatsby-plugin-image for some pictures?

If not, i think it can be considered as a bug/failure. Images should be preloaded not only for vertical, but also for horizontal threshold of a viewport - to support carousels, sliders, horizontally scrolled pages etc.

**Comment by wardpeet** [comment]
Hey, is it possible to share a small reproduction so I understand correctly what you're asking? At this moment you can't disable the transition right now.
